scripts/retort-sync.ps1

```
@@ -0,0 +1,144 @@
+        $changed  = (git -C $ProjectRoot diff --name-only 2>$null | Measure-Object -Line).Lines
+        $newFiles = (git -C $ProjectRoot ls-files --others --exclude-standard 2>$null | Measure-Object -Line).Lines
+        Write-Host '[retort-sync] Post-sync git summary:'
+        Write-Host "              Modified (tracked): $changed file(s)"
+        Write-Host "              New (untracked):    $newFiles file(s)"
+        if ($changed -gt 0 -or $newFiles -gt 0) {
+            Write-Host ''
+            Write-Host "              Run 'git diff --stat' to review changes."
+        }
+    } catch {
+        # Not a git repo or git failed — skip summary
+    }
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)